src/components/accessories/opds/filter/OpdFilterForm.tsx

```diff
@@ -83,27 +91,65 @@ export const OpdFilterForm: FC<IOpdFilterProps> = ({ fields, onSubmit }) => {
             ) <= 0
           );
         },
+      })
+      .test({
+        name: "isFuture",
+        message: t("opd.futuredatenotallow"),
+        test: function (value) {
+          if (!moment(value).isValid()) return true;
+          return differenceInSeconds(new Date(value), new Date()) <= 0;
+        },
+      }),
+    ageFrom: number()
+      .test({
+        name: "ageFrom",
+        message: t("opd.validatefromage"),
+        test: function (value) {
+          if (isEmpty(this.parent.ageTo)) {
+            return true;
+          }
+          return +this.parent.ageTo - +value >= 0;
+        },
+      })
+      .test({
+        name: "minAgeFrom",
+        message: t("opd.belowminage"),
+        test: function (value) {
+          return !value || (value && value >= 0);
+        },
+      })
+      .test({
+        name: "maxAgeFrom",
+        message: t("opd.abovemaxage"),
+        test: function (value) {
+          return !value || (value && value <= 200);
+        },
+      }),
+    ageTo: number()
+      .test({
+        name: "ageTo",
+        message: t("opd.validatetoage"),
+        test: function (value) {
+          if (isEmpty(this.parent.ageFrom)) {
+            return true;
+          }
+          return +value - +this.parent.ageFrom >= 0;
+        },
+      })
+      .test({
+        name: "minAgeFrom",
+        message: t("opd.belowminage"),
+        test: function (value) {
+          return !value || (value && value >= 0);
+        },
+      })
+      .test({
+        name: "maxAgeFrom",
+        message: t("opd.abovemaxage"),
+        test: function (value) {
+          return !value || (value && value <= 200);
+        },
```

Comment:
I think that we should change the name from "From" to "To"

Reply:
Absolutely. `Copy and paste` effect :)